bertytech/content/faq/9- Who Berty is for/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Who Berty is for? 
+id: 9
+private: true
+---
+
+Berty can be used by anyone to communicate in a secure way. But it would be especially useful for people and organizations that work with sensitive data: 
+
+**Military, government, enterprises, journalists, advocates, activists, universities.**
```

Comment:
Berty messenger and protocol are intended for anyone concerned about their freedom (and for those who are unaware that their freedom is under threat!).

In our world, anyone could have their data collected and their communications intercepted. This is the sad consequence of pervasive mass surveillance across the globe. 

Certain groups of people are at higher risk because of their activity: journalists, military personnel, government officials, activists, corporate members, lawyers, whistleblowers. Berty was designed with them in mind.

Reply:
I have updated the PR body.